### PR TITLE
RHDEVDOCS-5199: Correct the tile name in modules/op-uninstalling-the-…

### PR DESCRIPTION
**Pull request by eromanova97**
• **Aligned team**: Dev Tools
• **OCP version for cherry-picking**: `enterprise-4.10` and later
• **JIRA issue**:  [RHDEVDOCS-5199](https://issues.redhat.com/browse/RHDEVDOCS-5199)
• **Preview page**: [Uninstalling the Red Hat OpenShift Pipelines Operator](https://59194--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/uninstalling-pipelines.html#op-uninstalling-the-pipelines-operator_uninstalling-pipelines)
• **QE review**: Completed by @ppitonak 
• **Peer-review**: Completed by @shipsing

### CODE REVIEW

File: modules/op-uninstalling-the-pipelines-operator.adoc [hunk 1]
```diff
@@ -8,8 +8,9 @@
...
 .Procedure
-. From the *Operators* -> *OperatorHub* page, use the *Filter by keyword* box to search for `{pipelines-title} Operator`.
 
-. Click the *{pipelines-shortname} Operator* tile. The Operator tile indicates it is installed.
+. From the *Operators* -> *OperatorHub* page, use the *Filter by keyword* box to search for `{pipelines-title}` Operator.
 
-. In the *{pipelines-shortname} Operator* descriptor page, click *Uninstall*.
+. Click the *{pipelines-title}* Operator tile. The Operator tile indicates it is installed.
+
+. In the *{pipelines-title}* Operator descriptor page, click *Uninstall*.
```
Comment (shipsing) [inline modules/op-uninstalling-the-pipelines-operator.adoc:16]:
```suggestion
. In the *{pipelines-title}* Operator description page, click *Uninstall*.
```

**Review by shipsing — CHANGES_REQUESTED** [review]

**Comment by kalexand-rh** [comment]
@ekristova 4.9 is EOL. Unless this is critical information, please update the applicable versions to 4.10+

File: modules/op-uninstalling-the-pipelines-operator.adoc [hunk 2]
```diff
@@ -6,10 +6,18 @@
 [id='op-uninstalling-the-pipelines-operator_{context}']
 = Uninstalling the {pipelines-title} Operator
 
+You can uninstall the {pipelines-title} Operator in web console.
+
+[CAUTION]
+====
+Uninstalling only the Operator does not remove the {pipelines-shortname} default and optional components. If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
+====
```
Comment (ppitonak) [inline modules/op-uninstalling-the-pipelines-operator.adoc:14]: Why did you add this note? Something similar is already on the page so now it is there twice. I don't see much value.

File: modules/op-uninstalling-the-pipelines-operator.adoc [hunk 3]
```diff
@@ -6,10 +6,18 @@
...
+
 [discrete]
 .Procedure
-. From the *Operators* -> *OperatorHub* page, use the *Filter by keyword* box to search for `{pipelines-title} Operator`.
 
-. Click the *{pipelines-shortname} Operator* tile. The Operator tile indicates it is installed.
+. From the *Operators* -> *OperatorHub* page, use the *Filter by keyword* box to search for the {pipelines-title} Operator.
```
Comment (ppitonak) [inline modules/op-uninstalling-the-pipelines-operator.adoc:19]: Is it your convention to remove the special formatting for operator name here? It's OK if you are trying to refer to the product name but if the user is expected to type in this exact string, then the word `Operator` needs to be removed, otherwise the UI doesn't show any results. Maybe it would make sense to format it the same way as in following steps.

olm

File: modules/op-uninstalling-the-pipelines-operator.adoc (hunk 2 above)
Comment (ekristova) [reply modules/op-uninstalling-the-pipelines-operator.adoc:14]: @ppitonak  It was based on the peer review suggestion to add any warnings about what users should keep in mind before uninstalling. The idea is that some users might not read the full page from top to bottom. However, I agree with you that since this is pretty short chapter, maybe in this case it is okay to not mention it twice.

File: modules/op-uninstalling-the-pipelines-operator.adoc (hunk 3 above)
Comment (ekristova) [reply modules/op-uninstalling-the-pipelines-operator.adoc:19]: @ppitonak I did not catch this. Thanks!

File: modules/op-uninstalling-the-pipelines-operator.adoc [hunk 4]
```diff
@@ -8,8 +8,9 @@
```
[the remark this answers is not in the document]
Comment (ekristova) [reply modules/op-uninstalling-the-pipelines-operator.adoc:8]: Based on the conversation with QE, the warnings are not needed here. Therefore, I added a short one sentence as introduction.

**Review by shipsing — APPROVED** [review]
LGTM @ekristova

**Review by ppitonak — APPROVED** [review]

**Comment by ppitonak** [comment]
LGTM

**Review by abrennan89 — APPROVED** [review]

File: modules/op-uninstalling-the-pipelines-operator.adoc [hunk 5]
```diff
@@ -6,10 +6,13 @@
 [id='op-uninstalling-the-pipelines-operator_{context}']
 = Uninstalling the {pipelines-title} Operator
 
+You can uninstall the {pipelines-title} Operator in web console.
```
[the remark this answers is not in the document]
Comment (ekristova) [reply modules/op-uninstalling-the-pipelines-operator.adoc]: Good idea. Done.
